src/main/java/net/sf/jabref/model/entry/EntryUtil.java

```diff
@@ -57,4 +61,25 @@ public static String capitalizeFirst(String toCapitalize) {
         }
         return al;
     }
+
+    /**
+     * @param keywords a String of keywords
+     * @return an ArrayList containing the keywords. An emtpy list if keywords are null or empty
+     */
+    public static ArrayList<String> getSeparatedKeywords(String keywords) {
```

Comment:
I would move the method + constants in separate class because they should be kept together and are independent of all the other methods. Maybe create a class `Keywords`.